student.rb

```diff
@@ -1,11 +1,18 @@
 require './person'
 
 class Student < Person
+  attr_reader :classroom
+
   def initialize(classroom, age, name = 'Unkown')
     super(age, name)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.animals.push(self) unless classroom.student.include?(self)
```

Comment:
- [ ] Please refactor this line of code to:
  ```ruby
  classroom.students.push(self) unless classroom.students.include?(self)
  ```
  Let's do away with the animals, please.